XML/syntax_test_xml.xml

```diff
@@ -130,6 +130,8 @@
 <!--                           ^ punctuation.definition.tag.end -->
      text
 <!-- ^^^^ text -->
+     >
+<!-- ^ text -->
```

Comment:
a better test here would be `text - illegal - punctuation`, to avoid regressions - `text` will always be true due to the base scope, `text.xml`